components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -229,9 +229,10 @@
 <bnd_topo hgrid="10x15"     >atm/cam/topo/USGS-gtopo30_10x15_remap_c050520.nc</bnd_topo>
 
 <bnd_topo hgrid="ne4np4"   npg="0">atm/cam/topo/USGS-gtopo30_ne4np4_16x.c20160612.nc</bnd_topo>
-<bnd_topo hgrid="ne4np4"   npg="2">atm/cam/topo/USGS-gtopo30_ne4pg2_16xdel2-PFC-consistentSGH.c20190618.nc</bnd_topo>
+<bnd_topo hgrid="ne4np4"   npg="2">atm/cam/topo/USGS-gtopo30_ne4np4pg2_16x.c20160612_converted.nc</bnd_topo>
```

Comment:
This leads to BFB diffs.   Was this intended?

Reply:
We shouldn't be adding redundant topo files. Was the one that was already causing a problem?

Reply:
It was not intended -- I can remove that one.

Reply:
Don't remove it.  But I think we should rename it if this is the only place it appears.